src/commands/uploadMobile.js

```diff
@@ -0,0 +1,73 @@
+import { uploadMachO } from '../uploadMacho.js';
+import { uploadProguard } from '../uploadProguard.js';
+
+const PLATFORMS = {
+  IOS: 'ios',
+  ANDROID: 'android',
+};
+
+const PLATFORM_UPLOAD = {
+  [PLATFORMS.IOS]: uploadMachO,
+  [PLATFORMS.ANDROID]: uploadProguard,
+};
+
+export const command = 'upload-mobile <path(s..)>';
```

Comment:
I'm not overly familiar with the way `yargs` configures these commands, but it seems like this string controls how the values passed to the command functions are named - with currently the mobile upload commands fail for me because the parameter is being passed in as a value literally called `path(s...)`; but changing the string to `'upload-mobile <path>'` or `'upload-mobile <paths..>'` enables the ios and android uploads, respectively

Reply:
maybe we can use the `.implies` option to allow a list of paths for ios but require a single one for android?

Reply:
Another option would be to get [yargs.argv._[1]](https://yargs.js.org/docs/#api-reference) to get the passed param without needing to reference it by name